WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -122,12 +122,31 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
         sortingButton.sortingOption = sortingOption
 
         if optionChanged, reloadCards {
+            showGhost()
```

Comment:
Showing ghost animation when we are fetching new cards after user changes sorting option.